src/theme/components/searchBar.ts

```diff
@@ -14,6 +14,9 @@ const SearchBar = defineMultiStyleConfig({
       base: "column nowrap",
       md: "row nowrap",
     },
+    "#searchbar-textinput-searchBar-id-wrapper": {
```

Comment:
This works but only for the first example in the Storybook docs page. This should be more generic to target the wrapper for all inputs.